cmd/podman/main_local.go

```diff
@@ -25,8 +26,16 @@ import (
 const remote = false
 
 func init() {
-
-	rootCmd.PersistentFlags().StringVar(&MainGlobalOpts.CGroupManager, "cgroup-manager", "", "Cgroup manager to use (cgroupfs or systemd, default systemd)")
+	cgroupManager := ""
+	if runtimeConfig, err := libpod.DefaultRuntimeConfig(); err == nil {
+		cgroupManager = runtimeConfig.CgroupManager
+	}
+	cgroupHelp := "Cgroup manager to use (cgroupfs or systemd)"
+	if rootless.IsRootless() {
```

Comment:
Should we just hide the flag for rootless, like we do for `--latest` with remote?

Reply:
I think we will still need this for cgroup v2, that will happen through systemd

Reply:
How would I determine if I am rootless and running with cgroupv2?

Reply:
we have a function `IsCgroup2UnifiedMode()` to determine that